### ISSUE TITLE
etesync isn't optional

### ISSUE THREAD

**Issue by drzraf** [jrvarma/icalcli#3]
```
$ icalcli
Traceback (most recent call last):
  File "/home/x/.local/bin/icalcli", line 5, in <module>
    from icalcli.icalcli import main
  File "/home/x/.local/lib/python3.8/site-packages/icalcli/__init__.py", line 5, in <module>
    from icalcli.etesync_backend.etesync_crud import EtesyncCRUD
  File "/home/x/.local/lib/python3.8/site-packages/icalcli/etesync_backend/etesync_crud.py", line 3, in <module>
    import etesync as api
ModuleNotFoundError: No module named 'etesync'
```

**Comment by jrvarma** [comment]
I think that just dropping these two lines of imports from `__init__.py` will fix this issue. Those two imports can then be in `backend_interface` Will test this out. If that proves too difficult, `etesync` can be made a dependency.

**Comment by jrvarma** [comment]
Fixed in version 0.9.9 using a `try` block
